__tests__/lint/async2/redocly.yaml

```diff
@@ -0,0 +1,38 @@
+apis:
+  anyof:
+    root: ./anyof.yml
+  application-headers:
+    root: ./application-headers.yml
+  correlation-id:
+    root: ./correlation-id.yml
+  gitter-streaming:
+    root: ./gitter-streaming.yml
+  mercure:
+    root: ./mercure.yml
+  not:
+    root: ./not.yml
+  oneof:
+    root: ./oneof.yml
+  operation-security:
+    root: ./operation-security.yml
+  rpc-client:
+    root: ./rpc-client.yml
+  rpc-server:
+    root: ./rpc-server.yml
+  simple:
+    root: ./simple.yml
+  slack-rtm:
+    root: ./slack-rtm.yml
+  streetlights-kafka:
+    root: ./streetlights-kafka.yml
+  streetlights-mqtt:
+    root: ./streetlights-mqtt.yml
+  streetlights-operation-security:
+    root: ./streetlights-operation-security.yml
+  websocket-gemini:
+    root: ./websocket-gemini.yml
+
+extends: []
```

Comment:
We don't need to write this explicitly as Redocly CLI doesn't extend by default anymore when there is a config file. Just clarifying.

Reply:
Got it. I will remove it - and I it was in the guide as well, so this is a really good tip.